### ISSUE TITLE
More terse tabs component

### ISSUE THREAD

**Issue by mattrunyon** [deephaven/deephaven-plugins#273]
As a developer, I want a more terse tabs component so that I can more easily add tabs to my panels.

Currently requires

```python
ui.tabs(
  ui.tablist(
    ui.item('Item 1')
  ),
  ui.tabpanels(
    ui.item('Panel')
  )
)
```

Would prefer something like this where we map into the spectrum components client side.

```python
ui.tabs(
  ui.tab_panel(children, title="Panel 1")
)
```

**Comment by bmingles** [comment]
As part of this, we should think through how to simplify props for `ui.item` passed into tab_list and tab_panels:
* `key` - shouldn't be needed in cases where `on_selection_change` is not used. In cases where the user needs to identify which tab was selected programatically, matching `key` props have to be set on both the tab and the panel item.
* `text_value` - This needs to be set for a11y purposes on any `ui.item` that contains non-primitive children (e.g. icons). Items that have a single primitive child should already be able to derive the `text_value` automatically.

The main scenario that would be overly verbose would be something like:
```python
from deephaven import ui
from deephaven.plot import express as dx

stocks = dx.data.stocks()


@ui.component
def table_tabs(source):
    return ui.tabs(
        ui.tab_list(
            ui.item("Unfiltered", key="Unfiltered"),
            # Duplicate text, `key`, and `text_value`
            ui.item(ui.icon("vsGithubAlt"), "CAT", key="CAT", text_value="CAT"),
            ui.item("DOG", key="DOG"),
        ),
        ui.tab_panels(
            # Duplicate `key` and `text_value` for all of the panels
            ui.item(source, key="Unfiltered", text_value="Unfiltered"),
            ui.item(source.where("sym=`CAT`"), key="CAT", text_value="CAT"),
            ui.item(source.where("sym=`DOG`"), key="DOG", text_value="DOG"),
        ),
        flex_grow=1,
        aria_label="Tabs"
    )


tt = table_tabs(stocks)
```

One option would be to set a fallback value for the `key` prop to the `text_value` prop if one is provided, or vice versa.

**Comment by AkshatJawne** [comment]
Resolved by #489
